.circleci/config.yml

```diff
@@ -530,25 +542,42 @@ workflows:
             - code_quality
             - unit_tests-linux
             - e2e_tests-linux
-            - build
           filters:
             branches:
               ignore:
                 - master
-      - send_test_release:
+      - build:
+          name: build_test_release-linux
+          platform: linux
           requires:
             - send_test_release_approval
           filters:
             branches:
               ignore:
                 - master
+      - build:
+          name: build_test_release-mac
+          platform: mac
+          requires:
+            - send_test_release_approval
+          filters:
+            branches:
+              ignore:
+                - master
+      - send_test_release:
+          requires:
+            - build_test_release-linux
+            - build_test_release-mac
+          filters:
+            branches:
+              ignore:
+                - master
       - verify_valid_version:
           name: verify_valid_version_branch
           requires:
             - code_quality
             - unit_tests-linux
             - e2e_tests-linux
-            - build
```

Comment:
Why was build removed from here?

Reply:
The dependencies between jobs changed - now it looks like this:
```
verify_valid_version -> approval -> build -> release
```

whereas it looked like this:
```
verify_valid_version -> build -> approval -> release
```

Building is being tested with e2e tests, so there were no real reason to test building (with packaging) explicitly for each PR, that is why it will from now on only build and package after approval

Reply:
Okay, it does seem more cost efficient that way.